### PR TITLE
Update to more minimalist Toast designs

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Changed

- Update to more minimalist Toast designs

---

Ticket: https://jira.uitdatabank.be/browse/III-5226

Screenshot 2023-02-14 at 14 52 14
Screenshot 2023-02-14 at 14 52 11
Screenshot 2023-02-14 at 14 52 05

### CODE REVIEW

File: src/ui/Toast.tsx [hunk 1]
```diff
@@ -87,35 +66,43 @@ const VariantToStylesMap = {
...
   body: string;
   visible?: boolean;
   onClose?: () => void;
 };
 
-const Toast = ({ variant, visible, header, body, onClose }: Props) => {
+const Toast = ({ variant, visible, body, onClose }: Props) => {
+  const icon = useMemo(() => {
+    const icons = {
+      [ToastVariants.PRIMARY]: Icons.PLUS_CIRCLE,
```
Comment (brampauwelyn) [inline src/ui/Toast.tsx:77]: This should probably be an info icon? With a question mark?
Maybe we only need 3 toast variants: success, danger and warning?

**Review by brampauwelyn — APPROVED** [review]
Looks good 💅

File: src/ui/Toast.tsx [hunk 2]
```diff
@@ -87,35 +66,43 @@ const VariantToStylesMap = {
...
+        backgroundColor="transparent"
         color={getValue('textColor.dark')}
+        className={'d-flex justify-content-between flex-row'}
       >
-        {body}
+        <span className={'d-flex mr-2'}>
+          {icon && <Icon name={icon} className={`text-${variant} mr-2`} />}
+          {body}
+        </span>
+        {onClose && <Icon name={Icons.TIMES} onClick={onClose} />}
```
Comment (brampauwelyn) [inline src/ui/Toast.tsx:105]: Maybe decrease the size of the close button a bit, seems a bit big now?

File: src/ui/Toast.tsx [hunk 3]
```diff
@@ -87,35 +66,43 @@ const VariantToStylesMap = {
...
+      [ToastVariants.SUCCESS]: Icons.CHECK_CIRCLE,
+    };
+
+    return icons[variant];
+  }, [variant]);
+
   return (
     <BootstrapToast
-      className={`d-inline-block m-1 bg-${variant}`}
+      className={`d-inline-block m-1 p-2`}
```
Comment (brampauwelyn) [inline src/ui/Toast.tsx:88]: Maybe also add some box shadow on the toast.

File: src/ui/Toast.tsx (hunk 1 above)
Comment (Anahkiasen) [reply src/ui/Toast.tsx:77]: I'm not even sure danger and warning are used at the moment, the only variant in use I could find was success

File: src/ui/Toast.tsx (hunk 3 above)
Comment (Anahkiasen) [reply src/ui/Toast.tsx:88]: Not sure I get what you mean, the Bootstrap toasts already come with box shadow, or do you mean some inner box shadow?

Comment (brampauwelyn) [reply src/ui/Toast.tsx:88]: Aha ok it wasn't clear on the screenshot that there was box-shadow probably because of the grey background ;)

**Review by brampauwelyn — APPROVED** [review]

**Pull request merged**